python/paddle/fluid/incubate/fleet/base/role_maker.py

```diff
@@ -334,45 +334,34 @@ def __init__(self, is_collective=False):
     def generate_role(self):
         if not self._role_is_generated:
             if not self._is_collective:
-                self.port = os.getenv("PADDLE_PORT",
-                                      "6174")  # port of current server
-                self.pserver_ips = os.getenv("PADDLE_PSERVERS",
-                                             "")  # ip of server
+                port = os.environ["PADDLE_PORT"]
+                pserver_ips = os.environ["PADDLE_PSERVERS"]
```

Comment:
Shoud we print a warning message in case these environment variables are not exist or set a default value.

Reply:
what's the meaning if we give a default but fault value?

Reply:
So just print a warning message.

Reply:
done